cli/command_policy_set_retention.go

```diff
@@ -24,10 +25,11 @@ func (c *policyRetentionFlags) setup(cmd *kingpin.CmdClause) {
 	cmd.Flag("keep-weekly", "Number of most-recent weekly backups to keep per source (or 'inherit')").PlaceHolder("N").StringVar(&c.policySetKeepWeekly)
 	cmd.Flag("keep-monthly", "Number of most-recent monthly backups to keep per source (or 'inherit')").PlaceHolder("N").StringVar(&c.policySetKeepMonthly)
 	cmd.Flag("keep-annual", "Number of most-recent annual backups to keep per source (or 'inherit')").PlaceHolder("N").StringVar(&c.policySetKeepAnnual)
+	cmd.Flag("ignore-identical-snapshot", "Ignore identical snapshot with the last (or 'inherit')").StringVar(&c.policySetIgnoreIdenticalSnapshots)
```

Comment:
one last nit: `ignore-identical-snapshots` and update help to something like "Do not save identical snapshots"